File: Leetcode/3172-divisible-and-non-divisible-sums-difference/divisible-and-non-divisible-sums-difference.cpp
class Solution {
public:
    int differenceOfSums(int n, int m) {
        int sumN=0,sumM=0;
        for(int i=1;i<=n;i++){
           if(i%m!=0) sumN+=i; 
           else sumM+=i;
        }
        return (sumN-sumM);
        
    }
};

File: Leetcode/3172-divisible-and-non-divisible-sums-difference/divisible-and-non-divisible-sums-difference.py
class Solution(object):
    def differenceOfSums(self, n, m):
        sumN=0;
        sumM=0;
        for i in range(1,n+1,1):
            if i%m==0 :
                 sumM+=i;
            else :
                sumN+=i;
        return sumN-sumM;

        